apps/web/src/app/app/(dashboard)/[workspaceId]/monitors/_components/action-button.tsx

```diff
@@ -37,35 +37,32 @@ import {
   DropdownMenuTrigger,
 } from "@/components/ui/dropdown-menu";
 import { wait } from "@/lib/utils";
+import { api } from "@/trpc/client";
 
 type Schema = z.infer<typeof insertMonitorSchema>;
 
 interface Props {
-  // TODO: use type instead!
-  workspaceId: number;
-  url: string;
-  name: string;
-  description: string;
+  props: Schema;
 }
 
-// TODO: add correct types
-export function ActionButton({ workspaceId, ...props }: Props) {
+export function ActionButton({ props }: Props) {
```

Comment:
Feels double the wording. I'd suggest to use:

```suggestion
export function ActionButton(props: Schema) {
```

And update the props accordingly